src/App.tsx

```diff
@@ -1,61 +1,130 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+import { User } from './types/User';
+
+function getUser(userId: number): User {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+  const defaultUser = { name: '', email: '' };
+
+  return foundUser || defaultUser;
```

Comment:
it's either `null` or `user`
```suggestion
  return foundUser || null;
```